pgeth/engine.go

```diff
@@ -59,30 +53,19 @@ func NewEngine(cfg *PluginEngineConfig) *PluginEngine {
 }
 
 func (p *PluginEngine) Version(ctx context.Context) error {
-	pluginDirectory := os.Getenv("PGETH_DIRECTORY")
+	var plugins []*Plugin
 
-	if len(pluginDirectory) == 0 {
-		p.logger.Warn("Skipping plugin engine startup: PGETH_DIRECTORY is empty")
-		return nil
-	}
-
-	p.logger.Info("Starting plugin engine")
-
-	files, err := ioutil.ReadDir(pluginDirectory)
-	if err != nil {
-		return err
-	}
-
-	plugins := []*Plugin{}
-
-	for _, file := range files {
-		if file.IsDir() {
-			newPlugin, err := p.loadPluginDetailsFromDirectory(filepath.Join(pluginDirectory, file.Name()))
-			if err != nil {
-				return err
-			}
-			plugins = append(plugins, newPlugin)
+	// load monitoring plugin
+	if pluginMonitoringDirectory := os.Getenv("PGETH_MONITORING_CONFIG"); len(pluginMonitoringDirectory) != 0 {
```

Comment:
can we use `pluginConfigFile` like in `Start`

Reply:
mb